blockchains/stride/assets/staevmos/info.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "Stride Staked Evmos",
+    "type": "STRIDE",
+    "symbol": "STEVMOS",
+    "decimals": 6,
```

Comment:
18

Reply:
Is there something I need to do?or will the stride tokens show up automatically.thank u so much

Reply:
no, its all automatic after merge

Reply:
no, it's automatic after merge

Reply:
Thank you so much!!!!!!!